### ISSUE TITLE
Rename master branch to main

### ISSUE THREAD

**Issue by mrbean-bremen** [MeVisLab/pythonqt#144]
This came up a couple of times, and I personally am so used now to "main" being the default branch, that I constantly get confused.
The move itself is effortless, and GitHub provides some guidance as to how to change it locally if the move is done (I have done this a few times).

Please let us know, if you know of a reason not to do this rename, or to postpone it.

**Comment by iakov** [comment]
The only reason for me (but it is not a huge problem, frankly): some scripts (Dependabot, my own, Yocto/OE, maybe some distros) fetch sources by branch name. 
Thus, as a general rule: if there are no other reasons to change something, please, do not do it just for fun.

**Comment by mrbean-bremen** [comment]
Well, there is no really compelling reason for this, other than the reasoning behind the original renaming, and the fact that this is now the git standard for several years and many repos have been switching. I have done the switch for a few repos myself, so far without any problems. And yes, I often get confused calling `master` `main`, because I'm now so used to it...

Admittedly, this is not consistent inside the MeVisLab organization either - the two other actively maintained repos use `main`, while the (currently) inactive ones still use `master`.

**Comment by hjmjohnson** [comment]
@mrbean-bremen With the current changes that are going to need a new version number, no may be the best time to make this change.

**Comment by mrbean-bremen** [comment]
Agreed! Let me know when you (mainly yourself and @jcfr) are done with the changes, to avoid doing this while work is in progress.

**Comment by hjmjohnson** [comment]
@mrbean-bremen the switch from "master" -> "main" will not be a disruption for me.  I've dealt with it many times before.

I'll defer to @jcfr for the timing of the change.

**Comment by jcfr** [comment]
Since we will have to update the workflow to reference `main` instead of `master`, it could be done after https://github.com/MeVisLab/pythonqt/pull/313 is integrated 🚀 

If done before, it also will be straightforward to rebase.
